### PR TITLE
v1.4 UG and DG Documentation Fixes 

### PR DESCRIPTION
**Pull request by choowengyan**
These commits mainly fixes the documentation format across User Guide and Developer Guide.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -280,10 +281,10 @@ Format: `deletet INDEX t/TAG+`
 * Deletes one or more tags from a patient identified by the index number used in the last patient listing.
 * At least one tag must be provided.
 * Tags can only contain alphanumeric characters or spaces.
-* Tags cannot be empty or blank
+* Tags cannot be empty or blank.
 * Tags must be less than 50 characters long.
-* Tags are **case-insensitive** when matching tags
-* The index **must be a positive integer** 1, 2, 3, …​
+* Tags are **case-insensitive** when matching tags.
+* The index **must be a positive integer** 1, 2, 3, …​.
```
Comment (guohuang88) [inline docs/UserGuide.md]: The fullstop here is abit weird

**Review by guohuang88 — COMMENTED** [review]
Just a small change otherwise lgtm!

File: docs/UserGuide.md (hunk 1 above)
Comment (choowengyan) [reply docs/UserGuide.md]: removed! good catch!

**Review by guohuang88 — APPROVED** [review]
lgtm!

**Pull request merged**